.pylintrc

```diff
@@ -147,7 +147,7 @@ evaluation=10.0 - ((float(5 * error + warning + refactor + convention) / stateme
 [FORMAT]
 
 # Maximum number of characters on a single line.
-max-line-length=100
+max-line-length=200
```

Comment:
PEP8 requires a max line length of 79 characters tho

Reply:
.